### PR TITLE
Add Alternate Recipe for Coated Circuit Board

### PR DESCRIPTION
**Pull request by v3ect0rgames**

### CODE REVIEW

File: overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy [hunk 1]
```diff
@@ -3,6 +3,8 @@ import com.nomiceu.nomilabs.util.LabsModeHelper
 import static com.nomiceu.nomilabs.groovy.GroovyHelpers.RecyclingHelpers.*
 import static gregtech.api.GTValues.*
 
+
```
Comment (IntegerLimit) [inline overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy:6]: Remove these newlines.

File: overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy [hunk 2]
```diff
@@ -104,3 +106,11 @@ if (LabsModeHelper.expert) {
 		.buildAndRegister()
 
 }
+// Coated Circuit Board
```
Comment (IntegerLimit) [inline overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy:109]: Add a newline before the comment.

File: overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy [hunk 3]
```diff
@@ -104,3 +106,11 @@ if (LabsModeHelper.expert) {
...
+mods.gregtech.assembler.recipeBuilder()
+	.inputs(ore('plateWood'))
+	.fluidInputs(fluid('glue') * 150)
+	.circuitMeta(3)
+	.outputs(metaitem('board.coated'))
+	.duration(100).EUt(VA[ULV])
+	.buildAndRegister()
```
Comment (IntegerLimit) [inline overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy:116]: Add a newline at the end of the file.

**Review by IntegerLimit — CHANGES_REQUESTED** [review]

**Review by IntegerLimit — APPROVED** [review]

**Review by IntegerLimit — CHANGES_REQUESTED** [review]
Make the glue cost 100mb, and change the assembler recipe for 'Circuit Board' (the first one) from plate + glue + foil to coated board + foil.

File: overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy [hunk 4]
```diff
@@ -104,3 +104,21 @@ if (LabsModeHelper.expert) {
...
+	.fluidInputs(fluid('glue') * 100)
+	.circuitMeta(3)
+	.outputs(metaitem('board.coated'))
+	.duration(100).EUt(VA[ULV])
+	.buildAndRegister()
+
+mods.gregtech.assembler.removeByOutput([metaitem('circuit_board.basic')], null, null, null) 
+mods.gregtech.assembler.recipeBuilder()
+	.inputs(metaitem('board.coated'))
+	.inputs(item('gregtech:meta_foil', 25) * 4)
```
Comment (IntegerLimit) [inline overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy:120]: Here, use `.inputs(metaitem('board.coated'), ore('foilCopper') * 4)`

(Replacing Lines 119 and 120)

**Review by IntegerLimit — CHANGES_REQUESTED** [review]

File: overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy (hunk 4 above)
Comment (v3ect0rgames) [reply overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy]: re tested

**Review by IntegerLimit — APPROVED** [review]

**Pull request merged**